src/elements/content-sidebar/activity-feed/activity-feed/__tests__/ActivityThread.test.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { shallow, ShallowWrapper } from 'enzyme';
+import Comment from '../../comment/Comment';
+import AnnotationActivity from '../../annotations/AnnotationActivity';
+import ActivityThread from '../ActivityThread.js';
+import ActivityCard from '../../ActivityCard';
+
+describe('src/elements/content-sidebar/activity-feed/activity-feed/ActivityThread', () => {
+    // eslint-disable-next-line flowtype/no-types-missing-file-annotation
```

Comment:
We should try to avoid disabling linting rules whenever possible. Should this file be marked with a Flow annotation